docs/cdktf/python/resources/sleep.md

```diff
@@ -0,0 +1,140 @@
+---
+page_title: "time_sleep Resource - terraform-provider-time"
+subcategory: ""
+description: |-
+  Manages a resource that delays creation and/or destruction, typically for further resources. This prevents cross-platform compatibility and destroy-time issues with using the local-exec provisioner https://www.terraform.io/docs/provisioners/local-exec.html.
+---
+
+
+<!-- Please do not edit this file, it is generated. -->
+# time_sleep (Resource)
+
+Manages a resource that delays creation and/or destruction, typically for further resources. This prevents cross-platform compatibility and destroy-time issues with using the [`local-exec` provisioner](https://www.terraform.io/docs/provisioners/local-exec.html).
+
+-> In many cases, this resource should be considered a workaround for issues that should be reported and handled in downstream Terraform Provider logic. Downstream resources can usually introduce or adjust retries in their code to handle time delay issues for all Terraform configurations or upstream resources can be improved to better wait for a resource to be fully ready and available.
+
+## Example Usage
+
+### Delay Create Usage
+
+```python
+# Code generated by 'cdktf convert' - Please report bugs at https://cdk.tf/bug
+from constructs import Construct
+from cdktf import TerraformStack
+#
+# Provider bindings are generated by running `cdktf get`.
+# See https://cdk.tf/provider-generation for more details.
+#
+from imports.null.resource import Resource
+from imports.time.sleep import Sleep
+class MyConvertedCode(TerraformStack):
+    def __init__(self, scope, name):
+        super().__init__(scope, name)
+        # The following providers are missing schema information and might need manual adjustments to synthesize correctly: null.
+        #     For a more precise conversion please use the --provider flag in convert.
+        null_resource_previous = Resource(self, "previous")
```

Comment:
@DanielMSchmidt just wanted to double check after noticing this message, is this message okay?

Initially the code looks fine double checking with the generated `null_resource` docs: https://github.com/hashicorp/terraform-provider-null/blob/main/docs/cdktf/python/resources/resource.md#example-usage